src/libraries/Microsoft.Extensions.Hosting/src/HostingHostBuilderExtensions.cs

```diff
@@ -219,14 +219,23 @@ internal static void ApplyDefaultHostConfiguration(IConfigurationBuilder hostCon
                     new KeyValuePair<string, string?>(HostDefaults.ContentRootKey, cwd),
                 });
             }
+        }
 
+        internal static void AddDefaultHostConfigurationSources(IConfigurationBuilder hostConfigBuilder, string[]? args)
+        {
             hostConfigBuilder.AddEnvironmentVariables(prefix: "DOTNET_");
             if (args is { Length: > 0 })
             {
                 hostConfigBuilder.AddCommandLine(args);
             }
         }
 
+        private static void ApplyDefaultHostConfiguration(IConfigurationBuilder hostConfigBuilder, string[]? args)
```

Comment:
(super nit) I think this is a bit more readable if you move this method up to line 203, above the 2 internal static methods it calls.

Reply:
It must be the C developer in my that likes defining the callees first, but it's not like we need to avoid forward declarations. Updated.